cabal.project

```diff
@@ -27,57 +28,27 @@ constraints:
   hedgehog >= 1.0,
   bimap >= 0.4.0
 
-package Win32-network
-  tests: True
+tests: True
```

Comment:
Like in `cardano-node`: just enable all tests and disable those that we don't want (actually not needed anymore in Cabal 3.4! `source-repository-package`s are treated as real external packages instead of extra internal "source code". I'm keeping the code for the benefit of the users of 3.2.)

Now we can't forget to enable the tests of a newly introduced package.

Reply:
My current theory is that this change is causing the evaluation errors on CI:
```
in job ‘x86_64-w64-mingw32.shell.x86_64-linux’:
The Haskell package set does not contain the package: binary-search (build dependency).
If you are using Stackage, make sure that you are using a snapshot that contains the package. Otherwise you may need to update the Hackage snapshot you are using, usually by updating haskell.nix.
```
My idea comes from https://github.com/input-output-hk/cardano-node/blob/77d7e394fc7747114df8e8121a31afcabc91c06e/cabal.project#L46, although there they *disable* instead of *enable* the tests.

Reply:
Holy crap, that was it :roll_eyes:. I have to list each component explicitly for some reason. That was totally obvious from that error message :roll_eyes:.